samples/drivers/led_ws2812/boards/actinius_icarus_som_dk_nrf9160.conf

```diff
@@ -0,0 +1 @@
+CONFIG_WS2812_STRIP_SPI=y
```

Comment:
Is this a preparation for the removal of `WS2812_STRIP_DRIVER` (and `WS2812_STRIP_SPI` default) coming in the next patch of the series?

If yes, then I think you should merge these two patches.